src/styles/global.scss

```diff
@@ -5,4 +5,13 @@
   :global(.playkit-slideIn) {
     z-index: 1;
   }
+  :global(.playkit-container .playkit-video-filter) {
+    &:before {
+      display: none;
+    }
+
+    &:after {
```

Comment:
can the before and after be shorten? since they are the same

Reply:
yep